### PR TITLE
TH-892 | More fixes based on comments

### PR DESCRIPTION
**Pull request by Sartonon**
## Description :sparkles:

- Focus outline hidden fix on landing page
- Make other event time list item clickable from the whole area
- Fix back to top button not to focus skip link

## Additional notes :spiral_notepad:

### CODE REVIEW

**Review by nenonja — APPROVED** [review]
Approved, only thing I was thinking that when using keyboard (using tab + press enter) to click "back to top" it would work in the old way and show "siirry sisältöön" link, now keyboard user might have as "lost" feeling when going to top without visible focus. But that is really minor thing.

File: src/domain/app/resetFocus/ResetFocus.tsx [hunk 1]
```diff
@@ -16,7 +19,7 @@ const ResetFocus = (): React.ReactElement => {
     }
   }, [pathname]);
 
-  return <div ref={node} tabIndex={-1}></div>;
+  return <div ref={node} tabIndex={-1} id={resetFocusId}></div>;
```
Comment (nenonja) [inline src/domain/app/resetFocus/ResetFocus.tsx:22]: My IDE complains that tag is empty (so it could be `<div ... />`

Comment (Sartonon) [reply src/domain/app/resetFocus/ResetFocus.tsx]: fixed

**Comment by Sartonon** [comment]
> Approved, only thing I was thinking that when using keyboard (using tab + press enter) to click "back to top" it would work in the old way and show "siirry sisältöön" link, now keyboard user might have as "lost" feeling when going to top without visible focus. But that is really minor thing.

I think hitting tab before skip link makes sense (after back to top). That is how it works when opening the page or changing route.

**Pull request merged**